README.md

```diff
@@ -151,7 +151,7 @@ If not set, it will fall back to Prawn's default behavior.
 
 `details` and `line_items` allow inline formatting with Prawn. This allows you to use HTML tags to format text: `<b>` `<i>` `<u>` `<strikethrough>` `<sub>` `<sup>` `<font>` `<color>` `<link>`
 
-See [the Prawn docs](https://prawnpdf.org/api-docs/2.3.0/Prawn/Text.html#text-instance_method) for more information.
+See [the Prawn docs](https://prawnpdf.org/docs/prawn/2.5.0/Prawn/Text.html#text-instance_method) for more information.
```

Comment:
Maybe we should just link to Prawn's website so we don't have to update this every release?

```suggestion
See [the Prawn docs](https://prawnpdf.org/) for more information.
```

Reply:
I mainly wanted to fix the 404 error.
Now I see that it is the only link in the Readme that points to the Prawn docs, so it might be good to just link to the web page.